### ISSUE TITLE
Configuration of dart formatting not working

### ISSUE THREAD

**Issue by damolator** [eggnstone/DartFormatJetbrainsPlugin#20]
I've changed the formatting using the plugin but its not picking up the changes.   I've set the indent to 8 to test it, but its still an indent of 2.   (see image).

 I've also tried to use dart_format with json, but i'm getting an error.  Theres no documentation on the command line json format.  I've tried multiple ways of inserting quotes, double quotes, escaped quotes, no quotes, etc.  

  dart_format file.dart --config="{'IndentationSpacesPerLevel':3 }"
  Error in dart_format: FormatException: Unexpected character (at character 2)
  {'indentationspacesperlevel':3 }

Note: My OS is Windows 11, and i'm using Terminal when running commands

![image](https://github.com/eggnstone/DartFormatJetbrainsPlugin/assets/19574247/1e7b9274-d4b7-4528-a298-6c792a6ed489)

**Comment by eggnstone** [comment]
I split the JSON config out into a separate issue: #21 

For the formatting not working: I suspect that the built-in formatter formats right after my formatter.
Can you check that "format on save" is off?

![AndroidStudioFormatOnSave2](https://github.com/eggnstone/DartFormatJetbrainsPlugin/assets/11562425/cd69c9ee-83c8-4a21-8626-707c26166a62)

**Comment by staginghub** [comment]
Hi, 

Thanks for the quick reply.  I think I've found the issue, I'm using Android Studio 2023.2.1.   This is built on IntelliJ IDEA 2023.2 but doesn't seem to work with the dart_format plugin settings.    I've downloaded the latest IDEA version and its working, although i need to use Android Studio.

Heres the Android Studio checks:
The setting 'format on save' is off, and when I format it says 'Reformat: code successfully formatted '.  This suggests the plugin is working, but its not picking up the plugin settings.

**Comment by eggnstone** [comment]
I'll look into it.
But the message "reformat" is not from me, therefore I suspect Android Studio to interfere.

**Comment by staginghub** [comment]
Yes, there was a secondary keymap setup on Android studio. I’ve removed it and, unfortunately, the plugin doesn’t work at all.

**Comment by eggnstone** [comment]
I reset my Android Studio to the default settings (File -> Manage IDE settings -> Restore default settings because uninstalling and reinstalling kept my settings despite ticking the checkbox to delete my user settings).
Then I installed the Flutter plugin and, after that the DartFormat plugin.
Then I checked that "format on save" was off (it was already off).
`Crtl+Alt+,` works fine for me.
I searched the settings for "reformat" and it showed me the `Ctrl+Alt+L` for the built-in dart formatter.
When I use `Ctrl+Alt+L` I get the unwanted formatting:
![image](https://github.com/eggnstone/DartFormatJetbrainsPlugin/assets/11562425/c8fb7756-bfe5-4c7c-bdb5-130f28c4cb30)
But when I use `Crtl+Alt+,` I get the wanted formatting:
![image](https://github.com/eggnstone/DartFormatJetbrainsPlugin/assets/11562425/8de0e5a4-4c0d-4afd-ba20-6119cc581398)
I guess it still doesn't work for you?
What else could be different?

**Comment by damolator** [comment]
Sorry about the late reply, it was an Android studio installation/version issue.
